### ISSUE TITLE
Provide a Flatpak

### ISSUE THREAD

**Issue by LongJohn-Silver** [Radarr/Radarr#4551]
**Is your feature request related to a problem? Please describe.**
No, or the only problem would be the absence of a flatpak.

**Describe the solution you'd like**
Flatpaks are a more robust and secure alternative to snaps, and also is becoming a more standardized option for non-based and Ubuntu based distributions alike, being adopted already by twenty five flavours of Linux, and some of them come with flatpak preinstalled out of the box (CentOS, ElementaryOS, EndlessOS, Fedora, Linux Mint, PureOS and ZorinOS).

One of the biggest advantages in contrast with snaps, is that flatpaks are sandboxed, and therefore this results in a more secure and private experience since it is harder for some kind of malware to attack the system, or for said application to retrieve user information without them knowing it. Another good point is that everything that's uploaded to Flathub (Flatpak's centralized repository) is compiled by the same developers, and cannot be done by some random person on the internet which could lead to outdated packages, piracy, and whatnot. Just like snaps, Flatpaks are compatible with every distribution that supports it, so you don't have to be uploading a million different releases.

**Describe alternatives you've considered**
N/A

**Additional context**
More information:

https://flathub.org/
https://flatpak.org/
https://flatpak.org/setup/
https://docs.flatpak.org/en/latest/

**Comment by ta264** [comment]
Have you considered docker?

We don't provide an official docker image, I'm not sure we have any appetite to maintain an official flatpak that will see far less usage. Feel free to create one and upload it to whatever the equivalent of docker hub is :)

**Comment by LongJohn-Silver** [comment]
> Have you considered docker?
> 
> We don't provide an official docker image, I'm not sure we have any appetite to maintain an official flatpak that will see far less usage. Feel free to create one and upload it to whatever the equivalent of docker hub is :)

I have but I don't know if I have the technical skills to use Docker, whereas flatpaks are the easiest thing in the world. With regards to the usage, I don't if that's true, I can think of a lot of people that would use it if the installation is so much easier.

**Comment by Thirrian** [comment]
At a quick glance, flatpak looks like it's geared towards desktop applications, while docker is for services / "always-on" applications running in the background.
There are similarities but they fill different niches (I think), and docker is a better fit for sonarr/radarr/...?

**Comment by Qstick** [comment]
Agree with @ta264 I don't think this will happen officially given we don't care to maintain it, but feel free to fashion something up on your own to share with the community!

**Comment by LongJohn-Silver** [comment]
> At a quick glance, flatpak looks like it's geared towards desktop applications, while docker is for services / "always-on" applications running in the background.
> There are similarities but they fill different niches (I think), and docker is a better fit for sonarr/radarr/...?

I don't know, my computer is not that good and generally when I use Radarr I'm also using Bazarr and qBittorrent plus any other application I have open, which tends to slow down my machine, so I exit them after I have finished downloading the file, therefore it makes sense to me if it were released as a flatpak.



> Agree with @ta264 I don't think this will happen officially given we don't care to maintain it, but feel free to fashion something up on your own to share with the community!

As far as I know the developers of the software have to release their product on Flathub, no one else can, I think it's done that way to ensure that the packages are up to date (the snap for Sonarr is outdated, for example, because it was released by some person X), and to ensure that they do not contain malware and that it is not some sort of piracy (which I don't really care about, because I love piracy, but it's not a good idea to have a platform where anyone can upload any game, it will end up making it illegal and it would bring legal troubles). So, yeah, even if I had the skills to do this, I couldn't, I could create a flatpak to distribute on my own, but I don't know how to do that.
